packages/loader/container-loader/src/deltaManager.ts

```diff
@@ -439,10 +438,8 @@ export class DeltaManager
         // eslint-disable-next-line @typescript-eslint/no-floating-promises
         this._inboundSignal.pause();
     }
-
-    public dispose() {
-        throw new Error("Not implemented.");
-        this.isDisposed = true;
+    public dispose(error?: ICriticalContainerError | Error): void {
+        this.close(CreateContainerError(error));
```

Comment:
I'm not sure this is correct. In a sense of - who's going to call this method? I feel like IDeltaManager should not derive from IDisposable, there is closeFn that is passed to IRuntime to raise critical errors, runtime should not have another mechanism to raise critical errors to container (and there should be no way to dispose delta manager without closing container).

Reply:
i didn't mean to push this part of the change, slipped through my code moving around. but agree this need cleanup